astroid/interpreter/_import/spec.py

```diff
@@ -178,7 +178,7 @@ def contribute_to_path(self, spec, processed):
     ImpFinder,
     ZipFinder,
 )
-if _HAS_MACHINERY and sys.version_info[:2] > (3, 3):
+if _HAS_MACHINERY and sys.version_info[:2] > (3, 4):
```

Comment:
I think here we intended to have >= 3.4, this will stop working on 3.4

Reply:
Good point, will fix! (There was no test failure so here's a gap in coverage.)